modules/timepoint_list/php/module.class.inc

```diff
@@ -40,12 +40,19 @@ class Module extends \Module
      */
     public function handle(ServerRequestInterface $request) : ResponseInterface
     {
+        // FIXME This should be reworked to not silently handle the try-catch
         try {
             $resp = parent::handle($request);
             if ($resp->getStatusCode() != 404) {
                   return $resp;
             }
         } catch (\NotFound $e) {
+            /**
+             * This will suppress EmptyCatchBlock
+             * warnings in this method
```

Comment:
This warning type should just not be enforced by the phpmd config, it's a useless warning to have. Empty catch blocks are a perfectly valid thing to have.

Reply:
I'll disable this in the PHPMD config. For reference, here's the explanation.

> Usually empty try-catch is a bad idea because you are silently swallowing an error condition and then continuing execution. Occasionally this may be the right thing to do, but often it's a sign that a developer saw an exception, didn't know what to do about it, and so used an empty catch to silence the problem.

Reply:
There's a lot of assumptions in that explanation.